### DISCUSSION TITLE
what is difference in "latest version" ?

### DISCUSSION THREAD

**Discussion by safesummer** [xiaoyifang/goldendict-ng#485] [Q&A] [answered]
maybe version of xapian will work better than it? ANYWAY, I can't find any illustration for this

**Comment by shenlebantongying** [comment, marked as answer]
GD's homemade full text indexer is very slow compared to xapian, but the xapian in the "latest version" is new. Like all new software, it may contain bugs.

**Reply by safesummer** [reply]
thank you for replying,  I ought to have a try.
